app/views/content/help-and-support/_events.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= render(CallsToAction::Promo::PromoComponent.new(border: :bottom)) do |promo| %>
+  <% promo.left_section(
+      heading: "Online events",
+      link_text: "Find online events",
+      link_target: events_path({ teaching_events_search: { online: [true] }})) do %>
+    <p>Attend an online event to find out how to get into teaching.</p>
+  <% end %>
+  <% promo.right_section(
+        caption: "Events near you",
+        heading: "Events are a great place to ask questions") do %>
+      <p>In-person events are a great way to speak to teachers, local teacher training providers and expert advisers.</p>
+
+      <%= form_for TeachingEvents::Search.new, method: :post, url: events_path do |f| %>
+        <%= f.hidden_field :distance, value: 10 %>
+        <%= f.hidden_field :online, multiple: true, value: false %>
+        <%= f.label :postcode, "Enter postcode" %>
```

Comment:
@ethax-ross I got a 422 error on entering a postcode?

![image](https://user-images.githubusercontent.com/87642394/200260160-ee424bf5-b13b-4406-950c-db8811a0f5df.png)

Reply:
The PR was just way out of date with master and it was causing issues; I've merged in master and it looks good now